cmd/cluster-etcd-operator/main.go

```diff
@@ -27,7 +26,8 @@ func main() {
 	// overwrite gRPC logger, to discard all gRPC info-level logging
 	// https://github.com/kubernetes/kubernetes/issues/80741
 	// https://github.com/kubernetes/kubernetes/pull/84061
-	grpclog.SetLoggerV2(grpclog.NewLoggerV2(ioutil.Discard, os.Stderr, os.Stderr))
+	//grpclog.SetLoggerV2(grpclog.NewLoggerV2(ioutil.Discard, os.Stderr, os.Stderr))
+	grpclog.SetLoggerV2(grpclog.NewLoggerV2WithVerbosity(os.Stderr, os.Stderr, os.Stderr, 99))
```

Comment:
this is just making the etcd logs for the operator more verbose, simular to apiserver is now.